packages/matter.js/src/protocol/interaction/InteractionServer.ts

```diff
@@ -381,39 +415,48 @@ export class InteractionServer implements ProtocolHandler<MatterDevice> {
                 const { path, dataVersion } = values[0];
                 const attributes = this.endpointStructure.getAttributes([path], true);
```

Comment:
Hmm ok after reading through this bit maybe should be separate PR and/or on device-gen branch.

If the path is insufficiently concrete I think response should be INVALID_ACTION and processing should abort.  But if I'm reading correctly we'd just ignore here. 

I'm also wondering if logic could be more straightforward if we determine if the attribute exists then check to see if it's writtable. With the current flow I think we ask for the attribute then have to determine whether it wasn't returned because because it didn't exist or because it was read-only. 

Doing this on mobile so I may be reading incorrectly. Parents are sleeping in the office with my workstation. 😉

Reply:
Lol, all good, no need to hurry.

In fact we are again at:

if this getAttributes for one path returns nothing (like in this case) then this attribute bath does not exist! So it might be a valid "concrete path" but we do not have this cluster or attribute or endpoint ... This according to specs should return unsupportedWrite ...

But in fact you are right. the "validateConcrete*Path" methods should/could also check for the "invalidAction " cases ... they do not do that right now.

Reply:
OK, I already added the concrete path validation in the device-gen branch so maybe we merge this and we can look at simplifying during/after merge into device-gen

Reply:
I added a TODO